src/pages/VideoCall/VideoCall.js

```diff
@@ -4,7 +4,7 @@ import {SCREEN} from '../../styles/theme'
 
 export const CallContainer = styled.div`
     width: 100vw;
-    height: 100%;
+    height: 100vh;
```

Comment:
height needs to be back to 100%, as well as in html and body, otherwise the app is not well displayed on Android Chrome.

Reply:
I've just checked and did not have any issue on android chrome.

Reply:
I've replaced height by min-height to allow the content to be bigger though (was not needed on my resolution though)